QuickBytes/src/main/java/com/quickbytes/repository/RequestComboRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.quickbytes.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+
+import com.quickbytes.model.RequestCombo;
+
+public interface RequestComboRepository extends JpaRepository<RequestCombo,Long>{
+	
+	@Query("select rc from RequestCombo rc where rc.request.requestId=?1")
+	List<RequestCombo> getAllRequestCombosByRequestId(Long oid);
+
+	@Query("select rc from RequestCombo rc where rc.combo.comboId=?1")
+	List<RequestCombo> getAllRequestCombosByComboId(Long oid);
+
+	@Modifying
+	@Query("delete RequestCombo rc where rc.request.requestId=?1")
```

Comment:
Would this be "delete from RequestCombo..." instead of "delete RequestCombo...."

Reply:
No, from will cause an error in jpql unless you want to change to writing in native sql